BUILD.bazel

```diff
@@ -73,6 +84,25 @@ container_image(
     visibility = ["//visibility:public"],
 )
 
+container_image(
```

Comment:
I'm not 100% sure the `goarch` from `:bazel-remote-base-arm64` gets transfered to the `docker build` invocation. Can you please check?
I'm manually building with:
```bash
docker build --build-arg GOARCH=arm64 --tag bazel-remote-arm64 --file=Dockerfile .
```

Reply:
I made the same changes locally, then ran `bazel run //:bazel-remote-image-arm64`, then I extracted the container image to find `./app/bazel-remote-base-arm64.binary.runfiles/__main__/linux_arm64_static_pure_stripped/bazel-remote-base-arm64.binary` in one of the layers, which file confirms is the right format:
`ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, not stripped`

Reply:
Not sure. I've built on my PC and pushed to my own `hub` repo:
```bash
➜  bazel-remote git:(master) ✗ bazel run //:push_to_dockerhub-arm64
INFO: Analyzed target //:push_to_dockerhub-arm64 (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //:push_to_dockerhub-arm64 up-to-date:
  bazel-bin/push_to_dockerhub-arm64.digest
  bazel-bin/push_to_dockerhub-arm64
INFO: Elapsed time: 0.154s, Critical Path: 0.01s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
INFO: Build completed successfully, 1 total action
2020/02/29 13:53:06 Successfully pushed Docker image to index.docker.io/mihaigalos/bazel-remote-cache-arm64:latest
```
But when using it on the Pi, this happened:
```bash
➜  bazel-remote git:(master) docker run -v /home/mihai/bazel_cache:/data -p 9090:8080 -p 9092:9092 mihaigalos/bazel-remote-cache-arm64:latest --max_size=1
Unable to find image 'mihaigalos/bazel-remote-cache-arm64:latest' locally
latest: Pulling from mihaigalos/bazel-remote-cache-arm64
e8d8785a314f: Already exists 
c5d701115cb9: Already exists 
85cea451eec0: Already exists 
Digest: sha256:d984d94b92fc70c996a8ebebe1f06224b265dbef6a1f8343e20923dcc9b9ddc2
Status: Downloaded newer image for mihaigalos/bazel-remote-cache-arm64:latest
docker: Error response from daemon: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"/app/bazel-remote-base.binary\": stat /app/bazel-remote-base.binary: no such file or directory": unknown.
ERRO[0008] error waiting for container: context canceled
```

Reply:
Hmm, there is a `app/bazel-remote-base-arm64.binary` symlink inside the container. You should modify the new container_image to refer to this in the entrypoint.

Reply:
Awesome, works!